src/index.ts

```diff
@@ -94,7 +94,12 @@ program
       options.basePath = options.basePath || defaultBasePath;
     }
 
-    const ast = compileSolFiles(files, options);
+    // map file location relative to current working directory
+    const mFiles = files.map((file) => {
+      return path.relative(process.cwd(), file);
+    });
```

Comment:
I can be shortened by removing brackets and return
```suggestion
    const mFiles = files.map(file => path.relative(process.cwd(), file));
```

Reply:
great, I'll do